README.md

```
@@ -1,157 +1,111 @@
-  - [Documentation](./docs/figma/README.md)
-
-### Development Frameworks
-
-- **React/Tailwind**: Generate Tailwind configuration and React components
-  - [Documentation](./docs/tailwind/README.md)
-
-## Getting Started 🚀
-
-### Installation
+**Without Clafoutis:**
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)